ucsb-cs156-s24/proj-happycows-s24-4pm-5
# ISSUE: Fixed Delete Button on Announcements Page

## Overview

In this PR, I fixed the operation of the delete button on the Announcements Index Page. Now, once you click the delete button, it will remove the selected announcement and reload the page with the updated table.

## Merge Info
You must merge PR #48 first and then PR #56 before this PR because this PR depends on the fixes in the backend in 48 that solves the problem of announcements not appearing and depends on 56, where the fix for the edit/delete buttons not showing.

## Future Possibilities 

Some possible points could be:
- Right now, on delete, it will reload the page to update the table. Maybe in the future, on deletion it can update the table without reloading.

## Validation 

1. Go to my dokku link: https://team03-jasontnguyen-dev.dokku-05.cs.ucsb.edu
2. Go to list commons in the admin nav bar, then click on the Announcements button then try to delete an announcement
3. Use the swagger backend to get the id you just deleted to confirm it doesn't exist anymore.

## Tests


## Linked Issues

Closes #24
